exonum/src/runtime/mod.rs

```diff
@@ -122,8 +189,13 @@ where
     }
 }
 
+/// Useful artifact information for Exonum clients.
 #[derive(Debug, PartialEq)]
 pub struct ArtifactInfo<'a> {
+    /// List of protobuf files that make up the service interface, first element in tuple
+    /// is the file name, second is its content.
+    ///
+    /// The common interface entry point is always in `service.proto` file.
```

Comment:
Is it a restriction or just recommendation?

Reply:
Is it a restriction.